projects/swiot1l/src.mk

```diff
@@ -58,17 +58,17 @@ SRCS += $(DRIVERS)/api/no_os_gpio.c \
 INCS += $(DRIVERS)/adc-dac/ad74413r/ad74413r.h
 SRCS += $(DRIVERS)/adc-dac/ad74413r/ad74413r.c
 
-INCS += $(DRIVERS)/digital-io/max14906/max14906.h
-SRCS += $(DRIVERS)/digital-io/max14906/max14906.c
+INCS += $(DRIVERS)/digital-io/max149x6/max14906.h
```

Comment:
this is not enough, some .c files from this project need to be modified as well to use the max149x6 desc and init_param

Reply:
and i would merge this change with the commit where you added "base" functionality

Reply:
On it.

Reply:
Still not resolved